plugins/modules/sda_fabric_devices_workflow_manager.py

```diff
@@ -2581,8 +2586,8 @@ def get_device_params(self, fabric_id, network_id, device_details, config_index)
         if "LAYER_3" in border_types:
             if not (layer3_settings or have_layer3_settings):
                 self.msg = (
-                    "The parameter 'border_settings' is mandatory when the 'device_roles' has 'BORDER_NODE' "
-                    "for the device {ip}.".format(ip=device_ip)
+                    "The parameter 'layer3_settings' is mandatory under 'borders_settings' when the "
```

Comment:
Shall we change 'mandatory' to 'required' ? 

"The 'layer3_settings' parameter is required under 'borders_settings' when 'device_roles' includes 'BORDER_NODE' for device {ip}.".format(ip=device_ip)